spec/factories/invoices.rb

```diff
@@ -3,5 +3,9 @@
     association :customer
     association :merchant
     status { "shipped" }
+
+    after :create do |invoice|
+      create :transaction, invoice: invoice
+    end 
```

Comment:
this totally saved the day. Excited about using Factory Bot and Faker now that I understand it is useful and fun!